.github/workflows/deploy.yml

```diff
@@ -70,7 +70,7 @@ jobs:
       - name: Get Deployment URL
         id: deployment-url
         run: |
-          SUBDOMAIN=$([[ $GITHUB_REF = "refs/heads/master" ]] && echo "" || echo "preview-${{ inputs.tree_hash }}.")
+          SUBDOMAIN=$([[ $GITHUB_REF = "refs/heads/master" || $GITHUB_REF = "refs/heads/main" ]] && echo "" || echo "preview-${{ inputs.tree_hash }}.")
```

Comment:
Could potentially get the default branch as a workflow argument (defaulting to `main`)

Reply:
Yeah. I'll leave that as an optimization for now and add a comment.

Reply:
Actually, making this mandatory makes sense to me. https://github.com/pleo-io/pleo-spa-cicd/pull/27/commits/a2c56ad9bb4a3da7fd0b8626ba1844d51d1f2a0b

Reply:
Correction: we get this from the GitHub event already: `${{ github.event.repository.default_branch }}`